sonar-text-plugin/src/main/java/org/sonar/plugins/secrets/api/PostFilterFactory.java

```diff
@@ -49,9 +50,9 @@ public static Predicate<String> createPredicate(@Nullable PostModule post, @Null
     return postFilter;
   }
 
-  static Predicate<String> filterForPatternNot(String patternNot) {
+  static Predicate<String> filterForPatternNot(List<String> patternNot) {
     return candidateSecret -> {
-      Matcher matcher = Pattern.compile(patternNot).matcher(candidateSecret);
+      Matcher matcher = Pattern.compile(String.join("|", patternNot)).matcher(candidateSecret);
```

Comment:
We should somehow note in the description that patterns should be concatenable. For example, I'm not sure, how it will work if pattern like `["(?i)aaa", "(?i)bbb"]` will be automatically concatenated